src/App.css

```diff
@@ -13,7 +71,188 @@ nav ul {
   margin-right: auto;
 }
 
+.profile-icon {
+  border: 1px solid #e8e8e8;
+  padding: 7px 10px;
+  border-radius: 50%;
+  cursor: pointer;
+}
+
+.person-icon {
+  color: #0290ff;
+}
+
 .container {
   max-width: 85%;
   margin: 0 auto;
 }
+
+.book-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  margin-bottom: 20px;
+  border: 1px solid #e8e8e8;
+  padding: 30px;
+}
+
+.title-div li {
+  display: inline-block;
+  font-size: 14px;
+  color: #0290ff;
+  font-weight: lighter;
+  cursor: pointer;
+  font-family: 'Roboto Slab', serif;
+}
+
+.title-div button {
+  background-color: white;
+  border: none;
+  border-left: 2px solid #e8e8e8;
+  border-right: 2px solid #e8e8e8;
+  padding: 0 10px;
+  margin: 0 10px;
+  font-size: 14px;
+  color: #0290ff;
+  font-weight: lighter;
+  font-family: 'Roboto Slab', serif;
+  cursor: pointer;
+}
+
+.title {
+  margin-bottom: 8px;
+  font-family: 'Roboto Slab', serif;
+}
+
+.author {
+  font-family: 'Roboto Slab', serif;
+  margin-bottom: 15px;
+  font-size: 14px;
+  color: #0290ff;
+}
+
+.progress-section {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  /* width: 100%;
+  max-width: 200px; */
+  padding-right: 30px;
+  margin-right: 30px;
+
+  /* border: 1px solid chocolate; */
+}
```

Comment:
- [ ] Kindly remove unused codes from your codebase. It's a good practice to not have unused codes in your codebase as that will make your code clean and lean and professional. Please remove it so your codebase looks clean and professional 👍